### DISCUSSION TITLE
How to extract life history traits

### DISCUSSION THREAD

**Discussion by jlmorano** [James-Thorson-NOAA/FishLife#30] [General]
Hi Jim,
I'm wanting to extract some of the life history traits for a limited number of species. Generally, some of the factors listed in Table 1 of Thorson, Maureaud, Frelat, et al. 2023. In the FishBase_and_Morphometrics database, it appears there are variables such as "habitatbathymetric", "habitatbenthopelagic", "habitatreefassociated", but it's not clear to me what the values correspond to or if they are related to the categorical factors listed in Table 1. Could you please guide me?
thanks,
Janelle

**Comment by James-Thorson-NOAA** [comment]
Thanks for asking here, I'm always hoping that these Discussion threads can help others too :)

So running the code in the "Accessing values" vignette we get a table below.  Each categorical factor starts with base, and that value plus subsequent ones sum to one and correspond to the probability of taking a given value of a categorical factor (base is the base value for that factor, and you have to look it up in the Thorson et al. 2023)

![image](https://github.com/user-attachments/assets/cfd46d9f-a2da-42c4-8aa9-b931eacc312b)
